common/lib/capa/capa/inputtypes.py

```diff
@@ -232,7 +234,7 @@ def __init__(self, system, xml, state):
 
         # put hint above msg if it should be displayed
         if self.hintmode == 'always':
-            self.msg = self.hint + ('<br/>' if self.msg else '') + self.msg
+            self.msg = HTML('{hint}<br/>{msg}' if self.msg else '').format(hint=self.hint, msg=self.msg)
```

Comment:
@cahrens: I think the linter complains with concatenation and strings that include HTML like `<br/>`, so it may need to be done with all the `HTML()` calls.